homeassistant/components/tado/climate.py

```diff
@@ -233,6 +241,11 @@ def create_climate_entity(
     cool_max_temp = None
     cool_step = None
 
+    if tado.isX:
+        heat_min_temp = 5.0
+        heat_max_temp = 30.0
```

Comment:
I believe we have the generics set in the constants. Why not use them? Or X has a different behaviour?

Reply:
Oh, yes there are constants for that, I have not seen them while writing this... The max temp is different for TadoX so probably the best way would be to define constants for the TadoX min and max temps and use them here.

Reply:
Good approach.

Reply:
done in my latest commit :)